### PR TITLE
Add ABB Robots Package.

### PR DESCRIPTION
**Pull request by DavidMansolino**
To launch the simulation and nodes:
```
ros2 launch webots_ros2_universal_robot universal_robot_rviz.launch.py
```

To check the motion (please be careful with my car :wink: ):
```
ros2 action send_goal /follow_joint_trajectory control_msgs/action/FollowJointTrajectory "{
  trajectory: {
    joint_names: [A motor, B motor, C motor, D motor, E motor, F motor],
    points: [
      { positions: [3.02, -1.63, -1.88, 1.01, 1.51, 1.13], velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], time_from_start: { sec: 5, nanosec: 500 } },
      { positions: [-1.01, 0.38, -0.63, -0.88, 0.25, -1.63], velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], time_from_start: { sec: 6, nanosec: 500 } },
      { positions: [-1.01, 0.38, -0.63, -0.88, 0.25, 6.2], velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1], time_from_start: { sec: 50, nanosec: 500 } }
    ]
  }
}"
```

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: webots_ros2_abb/worlds/abb_irb4600.wbt [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    DEF WHEEL_SHAPE Slot {
+      type "vehicle wheel"
+      endPoint TeslaModel3Wheel {
+        physics Physics {
+          density 10
+        }
+      }
+    }
+  ]
+  name "solid(1)"
```
Comment (omichel) [inline webots_ros2_abb/worlds/abb_irb4600.wbt]: I would name them "wheel0" to "wheel6".

File: webots_ros2_abb/worlds/abb_irb4600.wbt [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        TeslaModel3Windows {
+        }
+        TeslaModel3FrontLights {
+        }
+        TeslaModel3Details {
+        }
+      ]
+    }
+  ]
+  name "tesla"
```
Comment (omichel) [inline webots_ros2_abb/worlds/abb_irb4600.wbt]: I would name them "tesla0" to "tesla2"

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**